docs/operator-guides/configuring-airbyte-db.md

```diff
@@ -87,5 +87,5 @@ The following command will allow you to access the database instance using `psql
 docker exec -ti airbyte-db psql -U docker -d airbyte
 ```
 
-To access the configuration files for sources, destinations, and connections that have been added, simply query the `airbyte-configs` table.
+Following tables are created `workspace`, `actor_definition`, `actor`, `actor_oauth_parameter`, `operation`, `connection`, `connection_operation`, `state`. These tables contain all the sources, destinations and connections information.
```

Comment:
do we have a list of what these tables correspond to? or is that in the tech spec you wrote?

can we either make the tech spec public readable and drop a link here, or leave a short description on each table's purpose? will be helpful to both OSS users and future airbyte folks.